packages/esm-form-entry-app/src/app/openmrs-api/obs-resource.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpParams } from '@angular/common/http';
+
+import { map } from 'rxjs/operators';
+import { WindowRef } from '../window-ref';
+import { fhirBaseUrl } from '@openmrs/esm-framework';
+
+@Injectable()
+export class ObservationResourceService {
+  constructor(protected http: HttpClient, protected windoRef: WindowRef) {}
```

Comment:
Any reason for `protected` instead of `private`?

Reply:
(I'm not seeing this being a parent class right now / in the future)

Reply:
No reason really, I have very limited knowledge of angular, so I was just following other files. I will change that to private.